lang/python/python-uci/Makefile

```
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2018-2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-uci
+PKG_VERSION:=0.8.1
+PKG_RELEASE:=2
+
+PYPI_NAME:=pyuci
+PKG_HASH:=9287fe41b427dc5c167592d429be48c1e6cfe276225681b1bdefddfe90d7e941
+
+PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
```

In OpenWrt, it is required to use real name and has email address connected to GitHub acc. So, you can be maintainer or put me as a maintainer.

```suggestion
PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
```